<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilTest</class>
 <widget class="QWidget" name="UtilTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>1073</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="FileSelect">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>文件选择</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="inputw" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>10</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Input Folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ctkPathLineEdit" name="PathLineEdit">
           <property name="filters">
            <set>ctkPathLineEdit::Dirs|ctkPathLineEdit::NoDot|ctkPathLineEdit::NoDotDot</set>
           </property>
           <property name="currentPath">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="id" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>10</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Current File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="idSpinBox">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="zybz">
     <property name="font">
      <font>
       <underline>false</underline>
      </font>
     </property>
     <property name="title">
      <string>主要病灶</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="lleft">
        <item>
         <layout class="QVBoxLayout" name="bblx">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="bblx_">
            <property name="text">
             <string>病变类型</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mbly">
            <property name="text">
             <string>磨玻璃影</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="shb">
            <property name="text">
             <string>实变</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="dmdy">
            <property name="text">
             <string>低密度影</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="xxy">
            <property name="text">
             <string>线性影</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="bbfb">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="bbfb_">
            <property name="text">
             <string>病变分布</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="zy">
            <property name="text">
             <string>中央</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="yzqgfb">
            <property name="text">
             <string>沿支气管分布</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="xmfj">
            <property name="text">
             <string>胸膜附近</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="bzdx">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="bzdx_">
            <property name="text">
             <string>病灶大小（单选）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="jiejie">
            <property name="text">
             <string>&lt;3cm结节</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="bankuai">
            <property name="text">
             <string>3-10cm斑块</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="dapian">
            <property name="text">
             <string>&gt;10cm大片</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="msx">
            <property name="text">
             <string>弥散性</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rright">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="bbsm">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="bbsm_">
            <property name="text">
             <string>病变数目（单选）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="danfa">
            <property name="text">
             <string>单发</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="duofa24">
            <property name="text">
             <string>2-4，多发</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="duofa5">
            <property name="text">
             <string>5以上，多发</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="msx_2">
            <property name="text">
             <string>弥散性</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="bbwz">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="bbwz_">
            <property name="text">
             <string>病变位置（单选）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="zs">
            <property name="text">
             <string>左上</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="zx">
            <property name="text">
             <string>左下</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ys">
            <property name="text">
             <string>右上</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="yz">
            <property name="text">
             <string>右中</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="yx">
            <property name="text">
             <string>右下</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="qtzx">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>其他征象</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="left_2">
        <item>
         <layout class="QVBoxLayout" name="wgbh">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="wgbh_">
            <property name="text">
             <string>微观变化</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="xyjgzh">
            <property name="text">
             <string>小叶间隔增厚</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="xwzbtmd">
            <property name="text">
             <string>细网状不透明度</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="qttz">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="qttz_">
            <property name="text">
             <string>其他特征</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="zbzzwdjj">
            <property name="text">
             <string>主病灶之外的结节</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="dmzc">
            <property name="text">
             <string>动脉增粗</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="zqgbzh">
            <property name="text">
             <string>支气管壁增厚</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="kqzqgz">
            <property name="text">
             <string>空气支气管症</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="right_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="qttz_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="qttz__">
            <property name="text">
             <string>其他特征</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="plsz">
            <property name="text">
             <string>铺路石征</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="yz_2">
            <property name="text">
             <string>晕征</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="fyz">
            <property name="text">
             <string>反晕征</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="xmzh">
            <property name="text">
             <string>胸膜增厚</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="xqjy">
            <property name="text">
             <string>胸腔积液</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="zgzh">
            <property name="text">
             <string>纵膈增厚</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="xzxy">
            <property name="text">
             <string>血坠效应</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ypzd">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>阅片诊断（单选）</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="ypzd0">
        <property name="text">
         <string>甲乙流/CAP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ypzd1">
        <property name="text">
         <string>新冠</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="zdbw">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>诊断把握（单选）</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="zdbw0">
        <property name="text">
         <string>0-猜测，无把握</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="zdbw1">
        <property name="text">
         <string>1-有一定依据的推测</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="zdbw2">
        <property name="text">
         <string>2-把握程度较高</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Save">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
